api/Port.go

```diff
@@ -276,3 +286,118 @@ func (vrsConnection VRSConnection) processUpdates(updates *libovsdb.TableUpdates
 	}
 	return nil
 }
+
+// AddPortToAlubr0 adds Nuage port to alubr0 bridge
+func (vrsConnection *VRSConnection) AddPortToAlubr0(intfName string, entityName string, entityUUID string) error {
```

Comment:
thinking out loud..  i am okay with passing the intfName directly as string but the other two i am thinking need to be passed as a struct. At a later point if we want to add one more argument to the function, we can pass it as part of structure. That will not the break the code of the users who are already using the API.